### PR TITLE
Fix rendering of workflow import snippet

### PR DESCRIPTION
**Pull request by simonbray**
See https://github.com/galaxyproject/training-material/pull/3163#issuecomment-1042072859

### CODE REVIEW

File: topics/computational-chemistry/tutorials/zauberkugel/tutorial.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ key_points:
 - Pharmacophore-based target prediction is an efficient and cost-effective method.
 contributors:
 - aurelienmoumbock
-- simonbray
```
Comment (simonbray) [inline topics/computational-chemistry/tutorials/zauberkugel/tutorial.md:20]: @aurelienmoumbock thanks for adding me here, but my contribution was pretty small. I would rather not be listed as a contributor, if that's ok for you.

**Comment by simonbray** [comment]
@bgruening can you approve if you have a spare minute?

File: topics/computational-chemistry/tutorials/zauberkugel/tutorial.md (hunk 1 above)
Comment (aurelienmoumbock) [reply topics/computational-chemistry/tutorials/zauberkugel/tutorial.md:20]: @simonbray I disagree 😐 Without your help I would not have figured out how to install RDconf or create the tutorial. Moreover, you polished the tutorial.

Comment (simonbray) [reply topics/computational-chemistry/tutorials/zauberkugel/tutorial.md]: Okay, then I will readd myself :)

**Review by hexylena — APPROVED** [review]

**Comment by simonbray** [comment]
Thanks @hexylena!

File: topics/computational-chemistry/tutorials/zauberkugel/tutorial.md [hunk 2]
```diff
@@ -323,20 +323,20 @@ The resulting `concatenated_scores` needs to be re-sorted according to the align
 
 For pharmacophore-based protein target prediction, you can choose to use Galaxy tools separately and in succession as described above, or alternatively use the one-step Zauberkugel workflow as described below (Figure 3).
 
-![Snapshot of Zauberkugel workflow]({% link topics/computational-chemistry/images/zauberkugel.png %} "Zauberkugel — protein target prediction of a bioactive ligand with Align-it and ePharmaLib")
-
 > ### {% icon hands_on %} Upload the Zauberkugel workflow
 >
->    {% snippet faqs/galaxy/workflows_import.md %}
```
Comment (hexylena) [inline topics/computational-chemistry/tutorials/zauberkugel/tutorial.md:330]: I'm 80% sure the issue was too much leading space, that should work.

Comment (hexylena) [reply topics/computational-chemistry/tutorials/zauberkugel/tutorial.md:330]: Edit: ah, yes, I see you've done that below and good, that works then!

Comment (simonbray) [reply topics/computational-chemistry/tutorials/zauberkugel/tutorial.md:330]: Yes, I assume so as well.

**Pull request merged**